src/directive.js

```diff
@@ -19,7 +19,7 @@
       template: '<span ng-class="{\'is-placeholder\': placeholder && !editingValue}" ng-style="{\'max-width\': \'inherit\'}" >' +
```

Comment:
The `transclude: true` line can be removed, since we no longer include ng-transclude in the directive template.